crates/forge/src/running.rs

```diff
@@ -67,18 +83,54 @@ pub(crate) fn run_from_test_case(
         footer.iter()
     );
     let (hints_dict, string_to_hint) = build_hints_dict(instructions.clone());
-    let core_cairo_hint_processor = CoreCairoHintProcessor {
-        runner: Some(runner),
-        starknet_state: StarknetState::default(),
-        string_to_hint,
-        run_resources: RunResources::default(),
+
+    let block_context = build_block_context();
+    let account_context = build_transaction_context();
+    let mut context = EntryPointExecutionContext::new(
+        block_context.clone(),
+        account_context,
+        block_context.invoke_tx_max_n_steps.try_into().unwrap(),
+    );
+    let test_selector = String::from("test_case").to_felt252();
+    let entry_point_selector = EntryPointSelector(StarkHash::new(test_selector.to_be_bytes())?);
+    let entry_point = CallEntryPoint {
+        class_hash: None,
+        code_address: Some(ContractAddress::from(0_u8)),
+        entry_point_type: EntryPointType::External,
+        entry_point_selector,
+        calldata: Calldata(Arc::new(vec![])),
+        storage_address: ContractAddress(patricia_key!("0x0")),
+        caller_address: ContractAddress::default(),
+        call_type: CallType::Call,
+        initial_gas: u64::MAX,
```

Comment:
Cannot `usize::MAX` be used here? Or is it expecting u64.

Reply:
It is expecting u64, rust is strongly typed so it wouldn't work otherwise